src/covid_model_seiir/ode_model/ode_process.py

```diff
@@ -96,15 +96,15 @@ def __init__(self, df,
         cases_threshold = 50.0
         start_date = date[df[col_cases] >= cases_threshold].min()
         idx_final = idx & (date >= start_date)
-        while not any(idx_final):
+        while not np.sum(idx_final) > 2:
             cases_threshold *= 0.5
             print(f'reduce cases threshold for {self.loc_id} to'
                   f'{cases_threshold}')
             start_date = date[df[col_cases] >= cases_threshold].min()
             idx_final = idx & (date >= start_date)
             if cases_threshold < 1e-6:
                 break
-        if np.sum(idx_final) < 2:
+        if np.sum(idx_final) <= 2:
```

Comment:
Look at this line now, I think we probably could remove line `107` to `110`. Since if when `cases_threshold<1e-6` in the while loop still not satisfy the condition, it should probably break.

Reply:
True.  Will do.